dvc/tree/azure.py

```diff
@@ -29,21 +30,77 @@ class AzureTree(BaseTree):
     def __init__(self, repo, config):
         super().__init__(repo, config)
 
+        self._account_url = None
         url = config.get("url", "azure://")
         self.path_info = self.PATH_CLS(url)
 
         if not self.path_info.bucket:
             container = self._az_config.get("storage", "container_name", None)
             self.path_info = self.PATH_CLS(f"azure://{container}")
 
+        self._get_string_token_credential(config)
+
+        if not (self._conn_str or self._credential):
+            from azure.identity import (
+                AzureCliCredential,
+                ChainedTokenCredential,
+            )
+
+            if not self._storage_account:
+                logging.warning(
+                    "You must enter a storgae account name for this auth flow"
```

Comment:
storgae -> storage 

(could you please open it in one of the IDEs that could check for typos?)

Reply:
Yeah, just added a spell check, good spot!